content/docs/manage/operations.md

```diff
@@ -4,34 +4,187 @@ enableTableOfContents: true
 
-Operations include:
-
-- `apply_config`: Applies a new configuration to a Neon object or resource. For example, creating, deleting, or updating  PostgreSQL users and databases initiates this operation.
-- `check_availability`: Checks the availability of a [compute endpoint](/docs/reference/glossary#compute-endpoint). This operation, performed by the [availability checker](/docs/reference/glossary#availability-checker), is a periodic load generated by the Control Plane that makes sure a compute endpoint is able to start and read and write data.
-- `create_compute`: Creates a compute endpoint. Creating a compute endpoint, a branch with an endpoint, or a Neon project initiates this operation. A Neon project is created with a read-write endpoint by default. For related information, see [Manage endpoints](/docs/manage/endpoints).
-- `create_timeline`: Creates a [branch](/docs/reference/glossary#branch). This operation occurs when creating a child branch or when creating a new project, which includes a `main` branch. For related information, see [Manage branches]/docs/manage/branches).
-- `delete_tenant`: Deletes the stored data when a Neon project is deleted.
-- `delete_timeline`: Deletes a branch. The terms _timeline_ and _branch_ are synonymous. The term _timeline_ is used internally in Neon.
-- `start_compute`: Starts a compute endpoint when there is an event or action that requires compute resources. For example, connecting to a suspended compute endpoint initiates this operation.
-- `suspend_compute`: Suspends a compute endpoint after a period of inactivity. For information about how Neon manages compute resources, see [Compute lifecycle](/docs/introduction/compute-lifecycle/).
+| Operation | Description  |
+|:---|:---|
+| `apply_config` | Applies a new configuration to a Neon object or resource. For example, creating, deleting, or updating  PostgreSQL users and databases initiates this operation. |
+| `check_availability`  | Checks the availability of a [compute endpoint](/docs/reference/glossary#compute-endpoint). This operation, performed by the [availability checker](/docs/reference/glossary#availability-checker), is a periodic load generated by the Control Plane that makes sure a compute endpoint is able to start and read and write data.  |
```

Comment:
Checks availability of the data in the branch, as it's now primarily connected to a branch, not endpoint. Yet, it involves starting the endpoint. So we check that compute can start on some particular branch. We also skip branches without endpoint

Reply:
Revised.